tal/render/util.py

```diff
@@ -1,11 +1,57 @@
+from dataclasses import dataclass
+
+import numpy as np
+
+
+@dataclass
+class Point2D:
```

Comment:
Just out of curiosity: is there any special advantage of using Point2D instead of just a tuple, apart from the nicer class name?

Seems fine for me, I'm just asking in case this would be useful in other parts of my projects

Reply:
Yeah, this is a bit controversial, initially I made it into a [named tuple](https://docs.python.org/3/library/collections.html#collections.namedtuple) along the lines of 
```
Point2D = namedtuple('Point2D', 'x y')
```
But then got an impression that it would be convenient to have numpy conversion methods directly on instances. But it doesn't seem to be the case, so it's probably better to revert to named tuples since they 
- would provide the convenience accessors like `pt.x`, 
- be compatible with the regular tuples, and
- require less code (literally a line per type :)